src/main/java/frc/robot/RobotContainer.java

```diff
@@ -97,7 +98,10 @@ private void configureAutoRoutines() {}
    * edu.wpi.first.wpilibj.Joystick} or {@link XboxController}), and then calling passing it to a
    * {@link JoystickButton}.
    */
-  private void configureButtonBindings() {}
+  private void configureButtonBindings() {
+    new JoystickButton(m_driverController, OIConstants.kY)
```

Comment:
Instead of using `kY` here, it would be better to use a named constant. For example, we currently have (obsolete, from the 2023 game) `k3rdRowButton` as an alias for `kY`. You can create a constant named something like `kTargetNote` and use that here. The main reason this matters is that it helps us quickly tweak controller key bindings once all the subsystems are integrated.